libs/coin-modules/coin-filecoin/src/api/api.ts

```diff
@@ -86,19 +91,50 @@ export const fetchEstimatedFees = makeLRUCache(
 );
 
 export const fetchBlockHeight = async (): Promise<NetworkStatusResponse> => {
-  const data = await fetch<NetworkStatusResponse>("/network/status");
-  return data as NetworkStatusResponse; // TODO Validate if the response fits this interface
+  const data = await fetch<NetworkStatusResponse>(currentVersion, "/network/status");
+  return data; // TODO Validate if the response fits this interface
 };
 
-export const fetchTxs = async (addr: string): Promise<TransactionResponse[]> => {
-  const response = await fetch<TransactionsResponse>(`/addresses/${addr}/transactions`);
-  return response.txs; // TODO Validate if the response fits this interface
+export const fetchTxs = async (
+  addr: string,
+  lastHeight: number,
+  offset: number,
+  limit: number,
```

Comment:
why are all options mandatory? could we have at least `limit` and `offset` with default values?